test/unit-tests/diagnostics.test.ts

```diff
@@ -404,6 +404,15 @@ suite('Diagnostics', async () => {
         expect(build_consumer.compilers.msvc.diagnostics[0].location.start.character).to.eq(0);
     });
 
+    test('Parse MSVC single proc error (older compiler)', () => {
```

Comment:
MSVC 2010 are more precise, I am not testing older compiler yet.

Reply:
MSVC 2010 is the older compiler.